python/ray/train/torch/config.py

```diff
@@ -14,6 +14,10 @@
 import torch
 import torch.distributed as dist
 
+from ray._private.ray_constants import HPU_PACKAGE_AVAILABLE
```

Comment:
For Train related changes, could you do it in a separate PR?

Reply:
Sure. Do you think, there will be more changes for train other than this to support all Ray functionalities?